frontend/src/App.js

```diff
@@ -210,6 +210,13 @@ export default function App() {
                   component={() => <Home />}
                   role=""
                 />
+                <SecureRoute
+                path="/NonconformityReportsByUnit"
+                exact
+                component={() => NonconformityReportsByUnit}
+                role="Reports" // Define the appropriate role
+              />
+
```

Comment:
Hey @Agaba-derrick  remove this route its of no use      `path="/RoutineReport"`  would be handling it !
 instead in 
`frontend/src/components/Reports/routine/Index.js` add ` {type === "patient" &&
              report === "haitiNonConformityBySectionReason" &&
              (< NonconformityReportsByUnit/>)}` cwhich is the comparison of params

Reply:
instead in
`frontend/src/components/Reports/routine/Index.js` add  `{type === "patient" && report === "haitiNonConformityBySectionReason" && (< NonconformityReportsByUnit/>)}` which is the comparison of params